python_tests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Installs the wheel compatible with CUDA 12 and cuDNN 8.9 or newer.
+# Note: wheels only available on linux.
+jax[cuda12_local]
```

Comment:
note that we do build against cuda11.7/cuda11.8 as well. So might not want to have a hard-coded cuda12 here.

if jax isn't needed for build, we can have the requirement listed in `setup.py`

Reply:
Yeah, but it should match the CUDA TK version on the system. ie. `jax[cuda11_local]` for cuda 11.7/8.

Reply:
Can you do this automatically in the pip `requirements.txt`? file

Reply:
Oops, sorry that I missed this comment.
I don't know if there's any way to do that in requirements.txt. But we are also not really using jax for build (unlike pybind & ninja), so it doesn't need to be installed with the requirements.txt (I think?!)

For pip wheel, we can have this as one of our `install_requires` (setup.py), which we will update in our pip building script.
For developers, well, we can just leave it out and ask people to install jax as a dependency. (maybe put this in the build wiki on github).

All I'm saying here is, let's just delete this file :laughing:

Reply:
I removed the `requirements.txt` file and added a `Pytest.md` file.